src/ET_Client.php

```diff
@@ -190,14 +197,43 @@ function __construct($getWSDL = false, $debug = false, $params = null)
             {
                 $this->scope = $params['scope'];
             }
+			if (array_key_exists('applicationType', $params))
```

Comment:
try to use only one style of writing the code

Reply:
i don't understand. can you be more specific?

Reply:
basically use thought you code either the syntax like this:
if (array_key_exists('applicationType', $params)) {
}

or as you wrote above:
if (array_key_exists('applicationType', $params))
{
}

The standard is the first one though